internal/globalconfig/globalconfig.go

```diff
@@ -62,3 +64,13 @@ func RuntimeID() string {
 	defer cfg.mu.RUnlock()
 	return cfg.runtimeID
 }
+
+// BoolEnv returns the parsed boolean value of an environment variable, or
+// def if it fails to parse.
+func BoolEnv(key string, def bool) bool {
```

Comment:
I don't know if this is the right package to hold this. Maybe `internal` ?

Reply:
Internal will work.